core/finality-grandpa/src/communication.rs

```diff
@@ -34,23 +34,23 @@ fn localized_payload<E: Encode>(round: u64, set_id: u64, message: &E) -> Vec<u8>
 }
 
 enum Broadcast<Block: BlockT> {
-	// set_id, round, encoded commit.
+	// round, set id, encoded commit.
```

Comment:
Do we maybe want to introduce an auxiliary unit struct `Round(u64)` and `SetId(u64)` to prevent incorrectly using them?